docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -457,30 +471,53 @@ Nonetheless, the providers effectively delegate most of the token-associated sta
 ====
 
 [[table]]
-.Token authentication mechanism comparison
+.Comparison of features by token authentication mechanism or extension
```

Comment:
I'd keep the original title, it is not really `or`

Reply:
Updated again.